.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version-file: 'go.mod'
+          go-version: '1.22'
```

Comment:
that is strange to me. xk6 is not a library, i do not expect it to be used by anybody. mb just have 1.21 in go mod? it is still supported and usually, such a version will have been tested for 6 month+
@roman-khimov

Reply:
or not

Reply:
We build binaries with the latest and greatest unless there is a reason to not do so. This PR just fixes an error made in #77 in fact.